src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -206,6 +206,7 @@ public int Millisecond
         /// Unix format string
         /// </summary>
         [Parameter(ParameterSetName = "net")]
+        [ValidateSetAttribute("FileDate", "FileDateUniversal", "FileDateTime", "FileDateTimeUniversal", AllowAll=true)]
```

Comment:
I would suggest using an `ArgumentCompleter` attribute instead - it's a bit more cumbersome, but if this scenario makes sense (and I think it probably does), we can find some way to make it simpler.

Reply:
Do you suggest enhance `ArgumentCompleter` to support `[ArgumentCompleter("FileDate", "FileDateUniversal", "FileDateTime", "FileDateTimeUniversal")] `?

Reply:
Just brainstorming here - but maybe a new attribute `ArgumentCompletions` that also implements `IArgumentCompleter` - this probably requires some additional code elsewhere, probably similar to the `ValidateSet` code for completion.

Reply:
A new attribute requires a lot of code. It seems easiest to add a new constructor `public ArgumentCompleterAttribute(params string[] completeStrings)`